test/integration/metrics/metrics_test.go

```diff
@@ -95,7 +100,6 @@ func TestMasterProcessMetrics(t *testing.T) {
 	checkForExpectedMetrics(t, metrics, []string{
 		"process_start_time_seconds",
 		"process_cpu_seconds_total",
-		"go_goroutines",
```

Comment:
The test was talking about processes specifically, I was too lazy to conditionally add the process metrics... could add this to the apiserver test below if we really want to confirm the presence of the goroutines metric?  Or can just add back in